samples/py_native/models/profiles.yaml

```
@@ -33,6 +33,7 @@ models:
         data:
           task: regression
           label_column: days_since_last_seen
+          label_value: 1
```

regression and label_value should throw an exception.

---

Removed from regression YAML